src/app/app.module.ts

```diff
@@ -4,7 +4,7 @@ import { BrowserModule } from '@angular/platform-browser'
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations'
 import { Router, RouterModule } from '@angular/router'
 import { MissingTranslationHandler, TranslateLoader, TranslateModule } from '@ngx-translate/core'
-
+import { PayloadNavigatedEvent } from '@onecx/integration-interface'
```

Comment:
NavigatedEventPayload

Reply:
libs were already released with this name.
Change it in the future?

Reply:
Please rename it and create a deprecated type acting as an alias 
```
@deprecated
type PayloadNavigatedEvent  = NavigatedEventPayload
```
With this, both are working but you are saying that `NavigatedEventPayload` is the variant you want to have in the future